Added logos to thirdparty libraries

Nyholm opened the pull request:
Related to https://github.com/php-http/documentation/issues/123

dbu commented:
:+1: 

i think having external images for those is fine. can you check if you get sphinx to not complain about that? i found http://stackoverflow.com/questions/12772927/specifying-an-online-image-in-sphinx-restructuredtext-format - maybe `suppress_warnings = ['image.nonlocal_uri']` is enough?

Nyholm commented:
Thank you for the review. 

Hm, `suppress_warnings = ['image.nonlocal_uri']` in conf.py did not work. I'll check this out later.. I have a meeting now.

Nyholm commented:
You were correct. I was running and old version of sphinx.

Nyholm commented:
I've also updated the path to the Swap logo. I added it here: https://github.com/php-http/httplug.github.io/pull/6

I think we should limit the number of logos here to six and then add more at httplug.io.

dbu commented:
thanks tobias!

agreed to limit to 6. if there are more, lets try to show the most known projects here.

Nyholm commented:
Awesome. 

Thank you for merging.
